src/wallet/rpcwallet.cpp

```diff
@@ -3602,7 +3602,7 @@ class DescribeWalletAddressVisitor : public boost::static_visitor<UniValue>
             UniValue subobj(UniValue::VOBJ);
             UniValue detail = DescribeAddress(embedded);
             subobj.pushKVs(detail);
-            UniValue wallet_detail = boost::apply_visitor(*this, embedded);
+            UniValue wallet_detail = std::visit(*this, embedded);
```

Comment:
Similar issue here (as above) using `std::visit`:
```bash
wallet/rpcwallet.cpp:3605:43: error: 'visit<const DescribeWalletAddressVisitor &, std::__1::variant<CNoDestination, PKHash, ScriptHash, WitnessV0ScriptHash, WitnessV0KeyHash, WitnessUnknown> &>' is unavailable: introduced in macOS 10.14
            UniValue wallet_detail = std::visit(*this, embedded);
                                          ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/variant:1534:26: note: 'visit<const DescribeWalletAddressVisitor &, std::__1::variant<CNoDestination, PKHash, ScriptHash, WitnessV0ScriptHash, WitnessV0KeyHash, WitnessUnknown> &>' has been explicitly marked unavailable here
constexpr decltype(auto) visit(_Visitor&& __visitor, _Vs&&... __vs) {
```

Reply:
resolved, now that a52ecc9 is merged